source/operations/install-deploy-manage/migrate-fs-gateway.rst

```diff
@@ -73,128 +72,143 @@ Procedure
    - Replace ``PATH`` with the IP address or hostname and port for the new deployment.
    - Replace ``ACCESSKEY`` and ``SECRETKEY`` with the credentials you used when creating the new deployment.
 
-#. Export the existing deployment's **configurations**
+#. Migrate settings according to the type of deployment:
 
-   Use the :mc-cmd:`mc admin config export <mc admin config export>` export command with the existing MinIO Client to retrieve the configurations defined for the existing standalone MinIO deployment.
+   .. tab-set::
 
-   .. code-block:: shell
-      :class: copyable
+      .. tab-item:: Gateway
 
-      mc admin config export ALIAS > config.txt
+         a. Migrate environment variables
 
-   - Use the existing MinIO Client.
-   - Replace ``ALIAS`` with the alias used for the existing standalone deployment you are retrieving values from. 
+            Copy the :ref:`environment variables <minio-server-environment-variables>` from the existing deployment's ``/etc/default/minio`` file to the same file in the new deployment.
```

Comment:
Is there a more accurate way to describe this?

The existing instructions suggest the old and new deployments could be on the same host. (Explicitly discusses using the same filesystem, which may or may not be the same host.) I couldn't figure that out, in part because then how do you have different `/etc/default/minio` files. (Maybe there's a flag to specify?)

Reply:
>I couldn't figure that out, in part because then how do you have different /etc/default/minio files
you could save the env in different files - it is just that the service file should refer the correct env file - see [here](https://min.io/docs/minio/linux/operations/install-deploy-manage/deploy-minio-single-node-single-drive.html#create-the-systemd-service-file)

```suggestion
            Copy the :ref:`environment variables <minio-server-environment-variables>` from the existing deployment's ``/etc/default/minio`` file to the EnvironmentFile specified in the new deployment's service file.
```

Reply:
So you can have more than one deployment if you are using systemd. (I'm not, my VMs have some sort of systemd problem.)

Manually incorporated your suggestion, thank you @poornas